dbbackup/management/commands/_base.py

```diff
@@ -0,0 +1,18 @@
+from optparse import make_option
+from django.core.management.base import BaseCommand, LabelCommand
+
+
+class BaseDbBackupCommand(LabelCommand):
```

Comment:
I think it's a better idea to keep the BaseCommand inheritance and make the change to LabelCommand in a separate issue where we can discuss API changes for the command line..